pylotoncycle/pylotoncycle.py

```diff
@@ -74,8 +74,7 @@ def GetWorkoutList(self, num_workouts=None):
             num_workouts = self.total_workouts
 
         limit = 100
-        pages = num_workouts // limit
-        rem = num_workouts % limit
+        pages = ( num_workouts // limit ) + ( num_workouts % limit > 0 )
```

Comment:
Small nit, mostly from the standpoint of readability.
How about keeping the original:
```
        pages = num_workouts // limit
        rem = num_workouts % limit
```
and then we can go:
```
if rem > 0:
    # There are still some remaining workouts, so add another page
    # to capture them all
    pages += 1
```